content/docs/examples/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Examples and tutorials"
```

Comment:
Please change the title to: Examples and Tutorials

The Kubeflow style guide recommends using title case for headings. In title case, almost every word is capitalized with the exception of little words like: and, in, or.